cmake/ThirdPartyPackagesKw.cmake

```diff
@@ -195,28 +195,26 @@ macro(build_openblas)
 
     set(OPENBLAS_CMAKE_ARGS
             ${EP_COMMON_CMAKE_ARGS}
-            -DVERSION=${OPENBLAS_VERSION}
             -DCMAKE_INSTALL_PREFIX=${KNOWHERE_INSTALL_PREFIX}
-            -DCMAKE_BUILD_TYPE=Release
             -DBUILD_SHARED_LIBS=ON
-            -DBUILD_STATIC_LIBS=ON
```

Comment:
static libs may be better for using.

Reply:
OpenBLAS depends on many other libs like omp, math, etc. And we can control linking behaviors when using dynamic libs. Or insist on using a static one?